console/ui/src/app/config/config.component.ts

```diff
@@ -153,29 +155,27 @@ export class ConfigComponent implements OnInit, OnDestroy {
   }
 
   public deleteData(): void {
-    this.deleteError = '';
-    this.deleting = true;
-    this.consoleService.deleteAllData('').pipe(delay(2000)).subscribe(
+    this.deleteConfirmService.openDeleteConfirmModal(
       () => {
-        this.deleting = false;
         this.deleteError = '';
-        this.deleteSuccess = true;
-      }, err => {
-        this.deleting = false;
-        this.deleteError = err;
+        this.deleting = true;
+        this.consoleService.deleteAllData('').pipe(delay(2000)).subscribe(
+          () => {
+            this.deleting = false;
+            this.deleteError = '';
+            this.deleteSuccess = true;
+          }, err => {
+            this.deleting = false;
+            this.deleteError = err;
+          },
+        );
       },
+      this.confirmDeleteForm,
+      'Delete Project (NOT Configuration)' ,
+      'Are you sure you want to delete all data?'
```

Comment:
```suggestion
     'Are you sure you want to delete all the database data?'
```
This should make it clearer, what do you think?